app/_data/schemas/frontmatter/concept.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
If we decide to require tiers, we might want to require them for concepts as well as how to. 

Ex. if we had a concept page about Advanced Analytics, that would be a paid tier.